pkg/model/crd.go

```diff
@@ -106,6 +107,16 @@ func (r *CRD) TypeRenames() map[string]string {
 	return r.sdkAPI.GetTypeRenames(r.cfg)
 }
 
+// Documentation returns the base documentation string for the API formatted as
+// a Go code comment block
+func (r *CRD) Documentation() string {
+	shape, ok := r.sdkAPI.API.Shapes[r.Names.Original]
+	if ok {
+		return shape.Documentation
+	}
+	return fmt.Sprintf("// %sSpec defines the desired state of %s", r.Names.Original, r.Names.Original)
+}
```

Comment:
I think that `{{ .Kind }}Spec defines the desired state of` should remain as first line comment. Maybe something like: 
```go
        docString := fmt.Sprintf("// %sSpec defines the desired state of %s\n", r.Names.Original, r.Names.Original)
        shape, ok := r.sdkAPI.API.Shapes[r.Names.Original]
        if ok {
		docString += shape.Documentation + "\n"
	}
	return docString
```